Examples/CodePushDemoApp-pre0.49/android/app/src/main/java/com/codepushdemoapp/MainApplication.java

```diff
@@ -30,7 +30,7 @@ public boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
           new MainReactPackage(),
-            new CodePush(getResources().getString(R.string.reactNativeCodePush_androidDeploymentKey), getApplicationContext(), BuildConfig.DEBUG)
```

Comment:
Just out of curiosity: Does anybody have historic context regarding why we used to prefix this string with `reactNativeCodePush_android`?

Reply:
That is good question. I did not find any real reason to keep this prefix. The IOS part has the `CodePush` prefix(`CodePushDeploymentKey`) which looks more consistent without any unnecessary context. And since we are adding new properties, it will be nice to have one prefix for all properties(like `CodePushDeploymentKey`, `CodePushPublicKey`, `CodePushServerUrl`).